### PR TITLE
ERMS-2780: Mailadressen von Teilnehmer in Umfragen werden nicht ausge…

### PR DESCRIPTION
**Pull request by Moetez16**
…geben

### CODE REVIEW

File: app/grails-app/controllers/com/k_int/kbplus/AjaxController.groovy [hunk 1]
```diff
@@ -2357,8 +2357,9 @@ class AjaxController {
     def getEmailAddresses() {
         Set result = []
         if (params.orgIdList){
-            List<Long> orgIds = params.orgIdList.split ','
+            List<Long> orgIds = (params.orgIdList.split( ',')).each { (it instanceof Long) ? it : Long.parseLong(it)}
             List<Org> orgList = orgIds.isEmpty() ? [] : Org.findAllByIdInList(orgIds)
+            println(orgList)
```
Comment (croqueGrec09) [inline app/grails-app/controllers/com/k_int/kbplus/AjaxController.groovy:2362]: debug residual?

File: app/grails-app/views/survey/_evaluationGeneralSurvey.gsp [hunk 2]
```diff
@@ -38,12 +38,9 @@
            value="${surveyResult.findAll { it.participant.hasAccessOrg() }}"/>
     <div class="four wide column">
     <g:if test="${surveyParticipantsHasAccess}">
-        <g:link data-orgIdList="${(surveyParticipantsHasAccess?.participant?.id).join(',')}"
-                    data-targetId="copyEmailaddresses_ajaxModal44"
-                    class="ui icon button right floated trigger-modal">
-            <g:message
-                    code="survey.copyEmailaddresses.participantsHasAccess"/>
-        </g:link>
+        <a data-semui="modal" class="ui icon button right floated" data-orgIdList="${(surveyParticipantsHasAccess.participant.id)?.join(',')}" href="#copyEmailaddresses_static">
```
Comment (croqueGrec09) [inline app/grails-app/views/survey/_evaluationGeneralSurvey.gsp:41]: a single id as list?

File: app/grails-app/views/survey/_evaluationGeneralSurvey.gsp [hunk 3]
```diff
@@ -169,12 +166,9 @@
 
     <div class="four wide column">
     <g:if test="${surveyParticipantsHasNotAccess}">
-        <g:link data-orgIdList="${(surveyParticipantsHasNotAccess?.participant?.id).join(',')}"
-                    data-targetId="copyEmailaddresses_ajaxModal55"
-                    class="ui icon button right floated trigger-modal">
-            <g:message
-                    code="survey.copyEmailaddresses.participantsHasNoAccess"/>
-        </g:link>
+        <a data-semui="modal" class="ui icon button right floated" data-orgIdList="${(surveyParticipantsHasNotAccess.participant.id)?.join(',')}" href="#copyEmailaddresses_static">
```
Comment (croqueGrec09) [inline app/grails-app/views/survey/_evaluationGeneralSurvey.gsp:169]: a single id as list?

File: app/grails-app/views/survey/_evaluationParticipantsView.gsp [hunk 4]
```diff
@@ -21,11 +21,9 @@
 
     <div class="four wide column">
     <g:if test="${surveyParticipantsHasAccess}">
-        <g:link data-orgIdList="${(surveyParticipantsHasAccess.participant.id).join(',')}"
-                data-targetId="copyEmailaddresses_ajaxModal2"
-                class="ui icon button right floated trigger-modal">
+        <a data-semui="modal" class="ui icon button right floated" data-orgIdList="${(surveyParticipantsHasAccess.participant.id)?.join(',')}" href="#copyEmailaddresses_static">
```
Comment (croqueGrec09) [inline app/grails-app/views/survey/_evaluationParticipantsView.gsp:24]: a single id as list? (and several times after)

File: app/grails-app/views/survey/_evaluationGeneralSurvey.gsp (hunk 2 above)
Comment (Moetez16) [reply app/grails-app/views/survey/_evaluationGeneralSurvey.gsp:41]: No. List of IDs.

File: app/grails-app/views/survey/_evaluationGeneralSurvey.gsp (hunk 3 above)
Comment (Moetez16) [reply app/grails-app/views/survey/_evaluationGeneralSurvey.gsp:169]: No. List of IDs.

File: app/grails-app/controllers/com/k_int/kbplus/AjaxController.groovy (hunk 1 above)
Comment (Moetez16) [reply app/grails-app/controllers/com/k_int/kbplus/AjaxController.groovy]: Ok. Thanks.

File: app/grails-app/views/survey/_evaluationGeneralSurvey.gsp (hunk 2 above)
Comment (croqueGrec09) [reply app/grails-app/views/survey/_evaluationGeneralSurvey.gsp:41]: Then the naming is strange. Where is surveyParticipantsHasAccess.participant being filled? It looks like a single id.

**Pull request merged**